ext/standard/mail.c

```diff
@@ -382,6 +381,11 @@ int wasix_sendmail(const char *host, uint16_t port, const char* username, const
 				   const char *headers, const char *subject, const char *mail_from, const char *mail_to, const char *data);
 #endif
 
+#ifdef __wasi__
+const char *username_env_var = "APP_MAIL_USERNAME";
```

Comment:
I'd use macros instead of constants here. We can do sth like:

```C
#define WASMER_EDGE_MAIL_USERNAME_VAR "APP_MAIL_USERNAME"
```

and put it up top next to `WASMER_EDGE_MAIL_POSTFIX`.

Reply:
Why do we call these WASMER_EDGE?

This is just generic configuration, wouldn't just `PHP_MAIL_...` be more sensible?

Reply:
Okay. Does `PHP_APP_MAIL_USERNAME` sound good?

Reply:
The edge var should be named: `MAIL_SMTP_USERNAME`.
Main reason for this is that the feature will enable SMTP emails for all apps, not just PHP.